Adjusting User guide structure (#42)
mckinsey/vizro: vizro-core/docs/pages/user_guides/parameters.md

# How to use parameters

This guide shows you how to add parameters to your dashboard. One main way to interact with the charts/components on your page is by changing the parameters of the underlying function that creates the chart/component.

> The [`Page`][vizro.models.Page] model accepts the `controls` argument, where you can enter a [`Parameter`][vizro.models.Parameter] model. If e.g. the charting function has a `title` argument, you could then configure a parameter that let's the dashboard user select the title of the chart via e.g. a dropdown.

Reviewer: ```suggestion
The [`Page`][vizro.models.Page] model accepts the `controls` argument, where you can enter a [`Parameter`][vizro.models.Parameter] model. If e.g. the charting function has a `title` argument, you could configure a parameter that allwows the user to select the chart title via e.g. a dropdown.
```

                  _target_: scatter
                  data_frame: iris
                  x: sepal_width
                  y: sepal_length
                  size: petal_length
                  color: species
                  color_discrete_map: {"setosa": "#00b4ff", "versicolor": "#ff9222"}
                id: scatter_chart
                type: graph
              - figure:
                  _target_: bar
                  data_frame: iris
                  x: sepal_width
                  y: sepal_length
                  color: species
                  color_discrete_map: {"setosa": "#00b4ff", "versicolor": "#ff9222"}
                id: bar_chart
                type: graph
            controls:
              - selector:
                  options: ["#ff5267", "#3949ab"]
                  value: #3949ab
                  multi: False
                  type: dropdown
                targets: ["scatter_chart.color_discrete_map.virginica", "bar_chart.color_discrete_map.virginica"]
                type: parameter
            title: My first page
        ```
    === "Result"
        [![Nested]][Nested]

    [Nested]: ../../assets/user_guides/control/control5.png

> In the above example, the object passed to the function argument `color_discrete_map` is a dictionary which maps the different flower species to fixed colors (eg. `{"virginica":"blue"}`). In this case we do not want to change the entire dictionary, but only the value `blue`. We do this by specifying a target as `scatter.color_discrete_map.virginica`.

Reviewer: ```suggestion
In the above example, the object passed to the function argument `color_discrete_map` is a dictionary which maps the different flower species to fixed colors (eg. `{"virginica":"blue"}`). In this case, only the value `blue` should be changed instead of the entire dictionary. This can be achieved by specifying a target as `scatter.color_discrete_map.virginica`.
```

> Note that in the above example we also have one parameter affect multiple targets.

Reviewer: ```suggestion
Note that in the above example, one parameter affect multiple targets.
```
